Observer/Admin/DashboardInstantiation.php

```diff
@@ -0,0 +1,158 @@
+<?php
+namespace NS8\Protect\Observer\Admin;
+
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Framework\App\ProductMetadataInterface;
+use Magento\Framework\Event\Observer;
+use Magento\Framework\Event\ObserverInterface;
+use Magento\Framework\Message\ManagerInterface;
+use Magento\Framework\Module\ModuleListInterface;
+use Magento\Framework\Registry;
+use Magento\Framework\UrlInterface;
+use Magento\Store\Model\StoreManagerInterface;
+use NS8\Protect\Exception\InstallException;
+use NS8\Protect\Helper\Config;
+use NS8\ProtectSDK\Installer\Client as InstallerClient;
+use NS8\ProtectSDK\Logging\Client as LoggingClient;
+
+/**
+ * Responds to Order Update events
+ */
+class DashboardInstantiation implements ObserverInterface
+{
+    /**
+     * Registry key used to determine fetching the access token between setup and update
+     */
+    const ACCESS_TOKEN_SET_KEY = 'ns8_access_token_set';
+
+    /**
+     * The  config manager
+     *
+     * @var Config $scopeConfig;
```

Comment:
Technically we don't need the variable name on these (especially when it's wrong). 😄